.vscode/settings.json

```diff
@@ -6,5 +6,9 @@
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
     "files.trimTrailingWhitespace": true,
-    "webgl-glsl-editor.format.placeSpaceAfterKeywords": true
+    "webgl-glsl-editor.format.placeSpaceAfterKeywords": true,
+    "rust-analyzer.check.extraArgs": [
+        "--target-dir",
+        "C:/windows/temp/rust-analyzer-check"
+    ],
```

Comment:
This doesn't look portable

Reply:
Yeah, I'm thinking I should just remove this file from the repo

Reply:
I'm not convinced the best way forward is to remove the whole file. These settings are good for keeping the files tidy and consistent. Can we at least resolve this in a separate PR?

Reply:
I guess the problem is that there's always going to be a case where a developer might have overrides that are unique to their specific environment, right? For me the rust analyzer one is super common that I need *per project* to avoid locking issues.

I think anything related to style etc should probably best be left to clippy, otherwise there's no way to, for example. get developers who aren't using VSCode to have their code correctly formatted.